crates/gitbutler-core/src/deltas/writer.rs

```diff
@@ -26,7 +26,7 @@ impl<'writer> DeltasWriter<'writer> {
         self.writer
             .write_string(PathBuf::from("session/deltas").join(path), &raw_deltas)?;
 
-        tracing::debug!(
+        tracing::trace!(
```

Comment:
I hope that by now it's fine to downgrade the severity of these low-level log message to make `debug` more useful for where it matters now. I assume that most `debug` traces will eventually be downgraded or downright removed as confidence (and test-coverage) grows.

Reply:
That's just fine :)